### PR TITLE
@craigspaeth Checks that href exists before trying to sanitize it

### PR DESCRIPTION
**Pull request by kanaabe**
This caused Positron to crash at 2:36PM today. `url.parse` doesn't fail gracefully so we should check that `$(this).attr 'href'` exists before trying to sanitize it. Weird that a link in the HTML didn't have an `href` field in the first place though.

### CODE REVIEW

**Comment by craigspaeth** [comment]
We should also definitely put [forever](https://github.com/foreverjs/forever) in our `make s` command like we do [on Force](https://github.com/artsy/force/blob/master/Makefile#L17). I'm kinda shocked it's taken this long to notice we need that—it's pretty standard for any Node production setup.

**Comment by craigspaeth** [comment]
Code looks good! However, I'd prefer we write a test for this to prevent a regression.

**Comment by kanaabe** [comment]
Added a test and make sf...oy it really is amazing how long we have not had this for.

**Comment by craigspaeth** [comment]
Solid! One last thing—make sure to add `make sf` [to the Procfile](https://github.com/artsy/positron/blob/master/Procfile#L1) so heroku acknowledges it as the command to run the production server with. Then good to merge!

**Comment by kanaabe** [comment]
Oh oops! Done.

File: Makefile [hunk 1]
```diff
@@ -12,6 +12,10 @@ BIN = node_modules/.bin
 s:
 	$(BIN)/coffee index.coffee
 
+# Start the server using forever
+sf:
+	$(BIN)/forever $(BIN)/coffee --nodejs --max_old_space_size=960 index.coffee
```
Comment (craigspaeth) [inline Makefile]: You can take the `--max_old_space_size=960` flag off here. That's an artifact of debugging the memory leak on Force. Luckily Positron averages less than a quarter of this memory use anyways so it wouldn't affect anything.

**Comment by craigspaeth** [comment]
Sorry :-P one more thing about the `max_old_space_size`... then it's good to merge!

**Comment by kanaabe** [comment]
Ok removed, merging!

**Comment by craigspaeth** [comment]
:+1:

**Pull request merged**